### ISSUE TITLE
About pytorch encoding and the training speed

### ISSUE THREAD

**Issue by irfanICMLL** [ansleliu/LightNet#16]
Hi,
Thank you for your great work! I am trying to do some distillation work on the small mobile net. I have some questions:
The code need the package pytorch encoding, but the pytorch encoding requires the pytorch to be 0.5.0. Your code is about pytorch 0.3.0+. I wonder how do you install the package of pytorch encoding?

However, the code can run under pytorch 0.5.0, but the training speed is very slow. I used a single P100 to have a try, and it need nearly 7 minutes to finish one epoch. If we need to train 10000 epochs, it will take nearly 48 days. 
Is this because the different pytorch version?

Thanks,
Yifan

**Comment by irfanICMLL** [comment]
Hello~ It is me again~ I still want to ask what version of pytorch  you use. Because before pytorch 0.4.0, the upsample operator is default to set the align corner=True. And with 0.4 and 0.5, align corner=False. I train from the pretrain model of mobile 
cityscapes_mobilenetv2_best_model.pkl , I set the align cornet= False. But I can only get 68% of mean IOU on the validation set.

Can you kindly share the details for this setting ?Or some other settings need to be notice during training?

Thank you very much!

**Comment by fip-lab** [comment]
> Hello~ It is me again~ I still want to ask what version of pytorch you use. Because before pytorch 0.4.0, the upsample operator is default to set the align corner=True. And with 0.4 and 0.5, align corner=False. I train from the pretrain model of mobile
> cityscapes_mobilenetv2_best_model.pkl , I set the align cornet= False. But I can only get 68% of mean IOU on the validation set.
> 
> Can you kindly share the details for this setting ?Or some other settings need to be notice during training?
> 
> Thank you very much!

hi, i try run this code, but not success, can you share version of depend?  Thank you very much!

**Comment by ozgurovic** [comment]
Hi, I just installed, so I wanted to share. You can use pytorch 0.3 with encoding v0.1.0. It works fine (at least for testing).
